alfresco-webscripts-client/alfresco-webscripts-ditto/src/integration-test/java/eu/xenit/alfresco/webscripts/client/ditto/SlingShotFakeClientIntegrationTest.java

```diff
@@ -19,4 +44,22 @@ public NodeLocatorClient nodeLocatorClient() {
         return new NodeLocatorFakeClient(dataSet);
     }
 
+    @Test
+    public void associationsTest() {
```

Comment:
I don't like that we have different tests for different implementations. Nevertheless I get why you needed to do this here. Having a look at possible improvements.

Reply:
We could put documents with association in the bootstrapped Alfresco in ditto?